src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -565,6 +565,11 @@
           "type": "string"
         },
 
+        "id": {
+          "description": "A unique identifer for the result.",
+          "type": "string"
+        },
+
```

Comment:
Tough to decide whether to put this first (typically things put their ID properties first, ID columns in databases tend to come first...) or whether to put it with things its most-likely-to-be-used-with, like `toolFingerprint`. Doesn't matter much... I wonder if there's a naming issue. We have `correlationId` for `run`. I wonder if this isn't just plain `id` but `externalDataLinkageId` or the like. Thoughts?